Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/CustomImageComparatorTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Aquality.Selenium.Template.NUnit.Constants;
+using Aquality.Selenium.Template.Utilities;
+using NUnit.Framework;
+using System.Drawing;
+
+namespace Aquality.Selenium.Template.NUnit.Tests
+{
+    public class CustomImageComparatorTest : BaseTest
+    {
+        const float TheThresholdValueOf10Percent = 0.1f;
```

Comment:
Constant name shall never contain it's numeric value

Reply:
The code was updated.